hibernate-reactive-core/src/main/java/org/hibernate/reactive/boot/impl/ReactiveServiceInitiators.java

```diff
@@ -0,0 +1,119 @@
+/* Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ * Copyright: Red Hat Inc. and Hibernate Authors
+ */
+package org.hibernate.reactive.boot.impl;
+
+import org.hibernate.boot.cfgxml.internal.CfgXmlAccessServiceInitiator;
+import org.hibernate.boot.registry.StandardServiceInitiator;
+import org.hibernate.bytecode.internal.BytecodeProviderInitiator;
+import org.hibernate.bytecode.internal.ProxyFactoryFactoryInitiator;
+import org.hibernate.cache.internal.RegionFactoryInitiator;
+import org.hibernate.engine.config.internal.ConfigurationServiceInitiator;
+import org.hibernate.engine.jdbc.batch.internal.BatchBuilderInitiator;
+import org.hibernate.engine.jdbc.connections.internal.MultiTenantConnectionProviderInitiator;
+import org.hibernate.engine.jdbc.cursor.internal.RefCursorSupportInitiator;
+import org.hibernate.engine.jdbc.dialect.internal.DialectFactoryInitiator;
+import org.hibernate.engine.jdbc.dialect.internal.DialectResolverInitiator;
+import org.hibernate.engine.jdbc.internal.JdbcServicesInitiator;
+import org.hibernate.engine.jndi.internal.JndiServiceInitiator;
+import org.hibernate.event.internal.EntityCopyObserverFactoryInitiator;
+import org.hibernate.jmx.internal.JmxServiceInitiator;
+import org.hibernate.persister.internal.PersisterFactoryInitiator;
+import org.hibernate.property.access.internal.PropertyAccessStrategyResolverInitiator;
+import org.hibernate.reactive.boot.impl.marker.ReactiveMarkerServiceInitiator;
+import org.hibernate.reactive.boot.service.NoJdbcConnectionProviderInitiator;
+import org.hibernate.reactive.boot.service.NoJdbcEnvironmentInitiator;
+import org.hibernate.reactive.boot.service.NoJtaPlatformInitiator;
+import org.hibernate.reactive.boot.service.ReactiveQueryTranslatorFactoryInitiator;
+import org.hibernate.reactive.id.impl.ReactiveIdentifierGeneratorFactoryInitiator;
+import org.hibernate.reactive.jpa.impl.ReactivePersisterClassResolverInitiator;
+import org.hibernate.reactive.pool.impl.SqlClientPoolInitiator;
+import org.hibernate.reactive.vertx.impl.VertxInstanceInitiator;
+import org.hibernate.resource.beans.spi.ManagedBeanRegistryInitiator;
+import org.hibernate.resource.transaction.internal.TransactionCoordinatorBuilderInitiator;
+import org.hibernate.service.internal.SessionFactoryServiceRegistryFactoryInitiator;
+import org.hibernate.tool.hbm2ddl.ImportSqlCommandExtractorInitiator;
+import org.hibernate.tool.schema.internal.SchemaManagementToolInitiator;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public final class ReactiveServiceInitiators {
+
+    private ReactiveServiceInitiators() {
+    }
+
+    public static final List<StandardServiceInitiator> LIST = buildInitialServiceInitiatorList();
+
+    private static List<StandardServiceInitiator> buildInitialServiceInitiatorList() {
```

Comment:
@gavinking , @DavideD this list is rather important; it's similar and yet has crucial differences compared to `org.hibernate.service.StandardServiceInitiators`.

@aguibert this list will need to be adapted within Quarkus..  you should be able to paste it in the Quarkus extension and then use it to override the default (this one) list? We certainly don't want to use the same list as Hibernate ORM "vanilla" or the quarkus-hibernate-orm extension.

I predict some pain to maintain this as we know have 4 different variations of this list - defined in different projects. Ideas welcome to improve on this?

The good news is that a mistake in these lists should fail catastrophically, so testing might be good enough to not make it too bad.